server/gql/models/students/customCreateResolver.js

```diff
@@ -0,0 +1,16 @@
+import { insertStudent, insertStudentSubjects } from '@server/daos/studentSubjects';
+import { transformSQLError } from '@server/utils';
+
+export const customCreateResolver = async (model, args, context) => {
+  try {
+    const res = await insertStudent(args);
```

Comment:
`insertStudent`, and `insertStudentSubjects` need to be a part of a transaction. If either one of them fails neither should get committed to the database